.github/workflows/CI.yml

```diff
@@ -31,6 +40,34 @@ jobs:
         secrets: |
           secret/data/products/connectors/ci/common DOCKERHUB_USER;
           secret/data/products/connectors/ci/common DOCKERHUB_PASSWORD;
+          secret/data/products/connectors/ci/common ARTIFACTORY_USR;
+          secret/data/products/connectors/ci/common ARTIFACTORY_PSW;
+
+    - name: Setup Java Build
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: '17'
+        cache: 'maven'
+        server-id: camunda-nexus
+        server-username: MAVEN_USERNAME
+        server-password: MAVEN_PASSWORD
+
+    - name: Deploy Maven Artifact
+      run: mvn -B compile generate-sources source:jar javadoc:jar deploy
+      if: github.ref == 'refs/heads/main'
+      working-directory: bundle/mvn/
+      env:
+        MAVEN_USERNAME: ${{ steps.secrets.outputs.ARTIFACTORY_USR }}
+        MAVEN_PASSWORD: ${{ steps.secrets.outputs.ARTIFACTORY_PSW }}
+
```

Comment:
Do we need to deploy this already? What are we using this for? If there is no need for this, let's not do this right now.
```suggestion
```

Reply:
Where is the problem to deploy a SNAPSHOT of something we plan to have? Better start doing it now and get into the rhythm? 
I think that you are right, that even a `install` should be sufficient as we need the Docker image right now, not yes the Maven artifact. I would still keep it as I find it correct :-)

Reply:
I commented it for now and created https://github.com/camunda/connectors-bundle/issues/31